packages/docs/concepts/references-and-templates.yaml

```diff
@@ -81,7 +85,7 @@ _ref:
 
             ## Nunjucks Templating
 
-            Templating can be taken further by referencing [Nunjucks](https://mozilla.github.io/nunjucks/) template files. If a file ends with the `.njk` file extension, the file will first be hydrated as a Nunjucks template, using the `vars` as template variables. If the file ends with `.yaml.njk` or `.json.njk`, the output of the template will then be parsed. Nunjucks templates are useful since the template file does not need to be valid yaml before it is hydrated, and features like for-loops and if-statements can be used.
+            Templating can be taken further by referencing [Nunjucks](https://mozilla.github.io/nunjucks/) template files by simply adding .njk to the file extensions, for example, .yaml.njk. Nunjucks templates hydrate variables at build time, before the yaml is parsed, enabling Nunjucks features like for-loops and if-statements.
```

Comment:
I think I prefer the original paragraph here, it feels like some details are lost here. Not sure, @SamTolmay, @JohannMoller  what do you guys think?

Reply:
I agree, let's stick with the previous version

Reply:
Ja same. The previous one was a bit more descriptive

Reply:
Change made in d9ad6e012a786eeb90ec2e938dd465040c158fb1